include/ceed/jit-source/cuda/cuda-jit.h

```diff
@@ -13,4 +13,11 @@
 #define CeedPragmaSIMD
 #define CEED_Q_VLA 1
 
+#define CEED_QFUNCTION_RUST(name)                                                                                                           \
+  static const char                             name##_loc[] = __FILE__ ":" #name;                                                          \
+  extern "C" CEED_QFUNCTION_ATTR __device__ int name##_rs(void *ctx, const CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out); \
+  CEED_QFUNCTION_ATTR static __device__ int     name(void *ctx, const CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out) {     \
+    return name##_rs(ctx, Q, in, out);                                                                                                  \
+  }
```

Comment:
```suggestion
#define CEED_QFUNCTION_RUST(name)                                                                                                           \
  extern "C" __device__ int name##_rs(void *ctx, const CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out); \
  static __device__ int     name(void *ctx, const CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out) {     \
    return name##_rs(ctx, Q, in, out);                                                                                                  \
  }
```
ok, I think I'm following now and I think this should be adequate?